includes/admin.form.inc

```diff
@@ -18,12 +18,15 @@
  */
 function presbydora_admin_form(array $form, array &$form_state) {
   $tableselect = presbydora_build_tableselect();
+  $default_values = variable_get('presbydora_access_bypass_pids', array());
+  $default_values = array_filter($default_values);
```

Comment:
The filtering here shouldn't be necessary: We [filter when saving](https://github.com/discoverygarden/presbydora/pull/14/files/3a28d4bccc2c1b7161a1a7c1fa4b61fab5297515..ee0ebc0191da957d82c74c1968050bb111f0d850#diff-0cdd19abe14e51fe9e7f061ec72c86d3R53)... The issue seen here was probably just because of the value in the variable from earlier in development (could've just deleted the `presbydora_access_bypass_pids` to fix it, I suspect).